modules/profile-store/pages/dotnet.adoc

```diff
@@ -291,6 +291,11 @@ For `FindActiveUsersByFirstName` it's a little trickier, because the N1QL is gen
 include::../examples/dotnet/UserProfileExample/UserProfileExample/Models/indexes.n1ql[tag=ix_findActiveUsersByFirstName]
 ----
 
+Creating the right indexes can be difficult. If you are having trouble creating the right index, here are two paths you can take to get help:
+
+* Starting in Couchbase Server 6.5, there is an index advisor that can automatically make index recommendations for you. To learn more about this, read the link:link:http://docs.couchbase.com/tutorials/profile-store/indexing.html[User Profile Indexing tutorial]. This tutorial uses the same type of data that you've already been creating if you've been following along with this User Profile Tutorial.
+* Check out the link:https://forums.couchbase.com/c/n1ql[N1QL category on the Couchbase Forums]. You can post a question about N1QL queries or indexes, and the community there can often help you.
```

Comment:
```suggestion
* Check out the https://forums.couchbase.com/c/n1ql[N1QL category on the Couchbase Forums]. You can post a question about N1QL queries or indexes, and the community there can often help you.
```

The AsciiDoc processor automatically handles text strings starting with `http`/`https` so you can drop the `link:` in those instances